apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -170,7 +173,7 @@ export const IncidentHints: React.FC<IncidentHintsProps> = (props) => {
             },
         })
 
-        return get(res, 'data.objs', [])
+        return res?.data?.incidentProperties?.filter(Boolean) || []
```

Comment:
best to use lodash get here

Reply:
Lodash will return you `any` in this case

Reply:
And in many other cases, so your frontend becomes less type-safe and bug-safe

Reply:
btw, you can also use `filter(nonNull)` to be even more type-safe